kustomizations/apps/data-hub-configs/elife-article-xml.config.yaml

```diff
@@ -8,6 +10,8 @@ elifeArticleXml:
           # parameters will be skipped if referenced file does not exist
           - parameterName: Authorization
             filePathEnvName: GITHUB_API_AUTHORIZATION_FILE_PATH
+      # XML elements that we are interested in (by replaceing '-' with '_')
+      searchedXmlElements: ['related_article', 'article_id', 'article_categories']
```

Comment:
I realised maybe another name could be good: https://github.com/elifesciences/data-hub-core-airflow-dags/pull/1251/files#r1274681260